pkg/services/validationService.go

```diff
@@ -193,6 +194,13 @@ func sendEvaluationResult(cliServiceClient *cliClient.CliClient, evaluationReque
 	var OSInfoFn = utils.NewOSInfo
 	osInfo := OSInfoFn()
 
+	nodesCount, nodesCountError := k8sMetadataUtil.GetNodesCount()
```

Comment:
maybe is better to put the init  `ClusterContext` outside from the sendEvaluationResult flow, to make mazgan of champs in the next collecting data